api/v1alpha1/k8ssandracluster_types.go

```diff
@@ -129,6 +140,53 @@ type EmbeddedObjectMeta struct {
 	Annotations map[string]string `json:"annotations,omitempty"`
 }
 
+// TODO Implement Stringer interface. It will helpful for debugging and testing.
+type CassandraConfig struct {
```

Comment:
I'd note that we are handling Cassandra config through an embedded struct, while Stargate config is currently an independent ConfigMap. I don't have any preference, but shouldn't we at least harmonize both to use the same approach?

Reply:
Cassandra configs are set in the `CassandraDatacenter.Spec.Config` property. The value is serialized into json and set in an environment variable in the config init container. cass-config-builder is responsible for generating the actual config files. While changing this has been discussed and something we will probably pursue at some point, it is definitely out of scope for the first alpha. This rules providing a ConfigMap for Cassandra configs for the time being.

@adutra @burmanm do we want to have Stargate also use cass-config-builder? On the one hand it gives us consistency. On the other hand, I would argue that it is low priority. Happy to discuss though.